src/workspace/config/snapshot.json

```diff
@@ -1,183 +1,110 @@
 {
   "snapshotDetails": {
-    "timestamp": "2023-06-20T12:17:19.360Z",
-    "runtimeVersion": "29.108.73.14",
+    "timestamp": "2022-01-10T11:26:57.741Z",
+    "runtimeVersion": "23.96.67.7",
```

Comment:
should this have been reverted all the way back to 23?

Reply:
it doesn't matter .. that was working before I updated the snapshot